src/containers/HomePage.tsx

```diff
@@ -53,89 +53,150 @@ const HomePage: React.FunctionComponent = () => {
   Config.indi.brandAmbassador.description = intl.get('indi-brand-ambassador-description');
   Config.indi.brandAmbassador.submit_button_text = intl.get('indi-brand-ambassador-submit-button-text');
 
-  return (
-    <ParallaxProvider>
-      <div className="home-page-component" data-region="viewPortRegion">
-        <Carousel />
-        <IndiRecommendationsDisplayMain render={['carousel', 'brand']} configuration={Config.indi} />
-        {/* eslint-disable-next-line no-return-assign */}
-        <div className="section section-3 container">
-          <div className="sub-section">
-            <img className="small-image" alt="home-espot-2" src={Config.siteImagesUrl.replace('%fileName%', homeEspot2FileName)} onError={(e: any) => { e.target.src = homeEspot3; }} />
-            <div className="text-block">
-              <span className="line line-1">
-                {intl.get('home-sub-espot-container2-first-line')}
-              </span>
-              <span className="line line-2">
-                {intl.get('home-sub-espot-container2-second-line')}
-              </span>
+  if (!Config.b2b.enable) {
```

Comment:
I'm wondering if this is the best way of presenting the b2b vs b2c homepage - strictly from a code cleanliness point of view. 
What if we had sub-components that are displayed in HomePage.tsx (B2BHomePage.tsx, B2CHomePage.tsx) and switch between those based on this condition, rather than having the large blocks returned? Let me know what you think 😄

Reply:
It is a great idea, will implement it